buildtest/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+    "editor": "vi",
+    "queues": [
+        {
```

Comment:
This config file answers my question that a "queue" is not necessarily a standard HPC queue, it's serving more as an executor, and should be called as such (and validated with subtypes similar to the Config File schemas).